src/shared/mesh/MeshBuilder.cpp

```diff
@@ -44,6 +44,20 @@ MeshBuilder MeshBuilder::HypercubeMesh( int dim, int n_els )
 
 MeshBuilder::MeshBuilder( mfem::Mesh&& mesh ) : mesh_{ std::move( mesh ) } { mesh_.EnsureNodes(); }
 
+MeshBuilder&& MeshBuilder::scale( std::initializer_list<double> scale_factors )
+{
+  SLIC_ERROR_ROOT_IF( static_cast<int>( scale_factors.size() ) != mesh_.SpaceDimension(),
```

Comment:
Do we want this to error out, or simply not scale the mesh and return a warning? I guess this will be used when creating tests so likely want to error out and strongly let the developer know they didn't scale the mesh in their test correctly.

Reply:
I'd say error out.  Warnings might be missed and lead to bugs.